src/Toml/Printer.hs

```diff
@@ -104,21 +124,25 @@ prettyKeyValue i = Text.concat . map kvText . HashMap.toList
             . formatTime defaultTimeLocale "%z"
 
 -- | Returns pretty formatted tables section of the 'TOML'.
-prettyTables :: Int -> Text -> PrefixMap TOML -> Text
-prettyTables i pref = Text.concat . map prettyTable . HashMap.elems
+prettyTables :: PrintOptions -> Int -> Text -> PrefixMap TOML -> Text
+prettyTables options i pref = Text.concat . map prettyTable . HashMap.elems
```

Comment:
I think that it's a good idea to apply `order` here as well. I think if you specify `shouldSort`, you should sort table names as well.